docs/lib/debugging/fragments/android/dev-menu/setup.md

```diff
@@ -0,0 +1,12 @@
+Run the following commands to add a copy of the file `amplify/.config/local-env-info.json` to the 
+directory `app/src/main/res/raw` in your Android project and rename the copy to `localenvinfo.json`:
+
+```bash
+cd [path to root directory of your Android project]
+cp amplify/.config/local-env-info.json app/src/main/res/raw
+mv app/src/main/res/raw/local-env-info.json app/src/main/res/raw/localenvinfo.json
+```
+
+If the amplify directory is not in your Android project folder, please follow the [project setup 
```

Comment:
This line should probably be its own "Note" rather than together in a paragraph with the next sentences.